### ISSUE TITLE
PP-Failure (Radarr & Sonarr)

### ISSUE THREAD

**Issue by mjumnito** [clinton-hall/nzbToMedia#1709]
**Describe the bug**
nzbtoMedia.py, nzbtoRadarr.py, nzbtoNZBdrone.py are not firing, to my knowledge, an update that allows media management (Radarr & Sonarr) to move and rename files associated with the downloaded movie or tv show

**Technical Specs**
1. Running on (Windows, Linux, NAS Model etc) windows
2. Python version '3.7'
3. Download Client: NZBget
4. Intended Media Management (SickChill, CouchPotoato, Radarr, Sonarr) 
Radarr in this instance but also does this in Sonarr. _Edit: added Log for Sonarr attempt as well_

**Expected behavior**
I expected nzbtoRadarr to trigger nzbtomedia to start the media management process. 
I also tried calling nzbtomedia to start the media management process but that failed as-well.

**Log**
### **Radarr**
info | Thu Jan 23 2020 11:39:03 | Collection Cabin.Fever.2002.720p.BluRay.x264-x0rEXTRA-Directors.Shorts,.The.Rotten.Fruit added to history
-- | -- | --
error | Thu Jan 23 2020 11:39:02 | Post-process-script  nzbToMedia\nzbToMedia.py for  Cabin.Fever.2002.720p.BluRay.x264-x0rEXTRA-Directors.Shorts,.The.Rotten.Fruit  failed (terminated with unknown status)
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: First error at line 82.
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: configobj.ConfigObjError: Parsing failed with several errors.
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     raise error
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\libs\common\configobj.py", line 1318, in _load
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     self._load(infile, configspec)
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\libs\common\configobj.py", line 1229, in __init__
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     super(configobj.ConfigObj, self).__init__(*args, **kw)
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\configuration.py", line 102, in __init__
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     cfg_new = config()
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\configuration.py", line 261, in addnzbget
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     CFG = config.addnzbget()
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\__init__.py", line 343, in configure_migration
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     configure_migration()
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\__init__.py", line 1019, in initialize
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     core.initialize(section)
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\nzbToMedia.py", line 885, in main
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:     exit(main(sys.argv))
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   File "E:\scripts\nzbToMedia\nzbToMedia.py", line 1052, in <module>
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Traceback (most recent call last):
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: -- Cleanup finished --
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Returning to directory:  C:\WINDOWS\system32
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: No folders to clean
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: -- Cleaning folders: [u'libs', u'core'] --
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Removing eol.pyc
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Removing cleanup.pyc
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Removing eol.pyc
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Removing cleanup.pyc
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: -- Cleaning bytecode --
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: Changing to directory: E:\scripts\nzbToMedia
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: [11:39:02] [ERROR]::MAIN: Unable to migrate config file E:\scripts\nzbToMedia\autoProcessMedia.cfg, exiting ...
info | Thu Jan 23 2020 11:39:02 | nzbToMedia:   warnings.warn(msg, LifetimeWarning)
info | Thu Jan 23 2020 11:39:02 | nzbToMedia: E:\scripts\nzbToMedia\eol.py:147: LifetimeWarning: Python 2.7 is no longer supported.
info | Thu Jan 23 2020 11:38:47 | Executing  post-process-script nzbToMedia\nzbToMedia.py for  Cabin.Fever.2002.720p.BluRay.x264-x0rEXTRA-Directors.Shorts,.The.Rotten.Fruit

### **Sonarr**

info | Thu Jan 23 2020 12:20:54 | Collection DCs.Legends.Of.Tomorrow.S05E02.INTERNAL.480p.X264-mSD added to history
-- | -- | --
error | Thu Jan 23 2020 12:20:54 | Post-process-script  nzbToMedia\nzbToMedia.py for  DCs.Legends.Of.Tomorrow.S05E02.INTERNAL.480p.X264-mSD failed (terminated  with unknown status)
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: First error at line 82.
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: configobj.ConfigObjError: Parsing failed with several errors.
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     raise error
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\libs\common\configobj.py", line 1318, in _load
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     self._load(infile, configspec)
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\libs\common\configobj.py", line 1229, in __init__
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     super(configobj.ConfigObj, self).__init__(*args, **kw)
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\configuration.py", line 102, in __init__
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     cfg_new = config()
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\configuration.py", line 261, in addnzbget
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     CFG = config.addnzbget()
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\__init__.py", line 343, in configure_migration
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     configure_migration()
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\core\__init__.py", line 1019, in initialize
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     core.initialize(section)
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\nzbToMedia.py", line 885, in main
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:     exit(main(sys.argv))
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   File "E:\scripts\nzbToMedia\nzbToMedia.py", line 1052, in <module>
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Traceback (most recent call last):
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: -- Cleanup finished --
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Returning to directory:  C:\WINDOWS\system32
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: No folders to clean
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: -- Cleaning folders: [u'libs', u'core'] --
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Removing eol.pyc
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Removing cleanup.pyc
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Removing eol.pyc
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Removing cleanup.pyc
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: -- Cleaning bytecode --
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: Changing to directory: E:\scripts\nzbToMedia
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: [12:20:54] [ERROR]::MAIN: Unable to migrate config file E:\scripts\nzbToMedia\autoProcessMedia.cfg, exiting ...
info | Thu Jan 23 2020 12:20:54 | nzbToMedia:   warnings.warn(msg, LifetimeWarning)
info | Thu Jan 23 2020 12:20:54 | nzbToMedia: E:\scripts\nzbToMedia\eol.py:147: LifetimeWarning: Python 2.7 is no longer supported.
info | Thu Jan 23 2020 12:20:50 | Executing post-process-script nzbToMedia\nzbToMedia.py for DCs.Legends.Of.Tomorrow.S05E02.INTERNAL.480p.X264-mSD

**Comment by clinton-hall** [comment]
Did you resolve this or was it closed in error?

Issue looks to be with migrating the config, so could be permissions on the .cfg file?, or some parameter being passed from NZBGet is the wrong type (e.g. used letters when it wanted a numeric input).

**Comment by mjumnito** [comment]
Yes I resolved this. I looked to me like the nzbtoMedia made 2 empty categories similar to the following in the autoProcessMedia.cfg file

>     [[""]]
>        host = localhost
>        apikey = ""
>        port = 5050
>        ssl = 0
>        web_root = ""
>        delete_failed = 0
>        wait_for = 6
>        remote_path = 0
>        enabled = 1

it made these at line 82 and another location farther down the file ( I think about 176 but I changed and saved before i noted the line number. To correct this, for whoever my find this in the future. I shut down NZBGet completely and removed the "empty category". Save the file and restart NZBGet. Upon restarting I did a post-process again and it worked.

I 'm not sure why these "empty categories" were created, but I remember I deleted the text of the cpsCategory of couchpotato via the web ui in the nzbtoMedia. I did the same further down the web-ui, maybe that had something to do with it. The file permissions were correct. Thanks for all the work that you have done on this awesome add-on.
